### PR TITLE
Add Netlify (Topics)

### PR DESCRIPTION
**Pull request by haroldao**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Because the topic "Netlify" is empty...

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/netlify/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: Netlify
+github_url: https://github.com/netlify
+logo: netlify.png
+short_description: Netlify: Develop & deploy the best web experiences
+topic: netlify
+url: https://netlify.com
+---
+
+Nelify is a powerful serverless platform with an intuitive git-based workflow.
```
Comment (BlakeWilliams) [inline topics/netlify/index.md:10]:
```suggestion
Netlify is a powerful serverless platform with an intuitive git-based workflow. 
```

**Comment by cgwizard796** [comment]
Nothing happen

**Comment by haroldao** [comment]
> Nothing happen

?

**Comment by krishdevdb** [comment]
I think that there are some issues maybe I can republish it with everything fixed or send the original author a pull request. I am creating a [branch](https://github.com/krishdevdb/explore/tree/netlify-topic) in my [fork](https://github.com/krishdevdb/explore).

**Comment by runningfromcyberstalker** [comment]
Please!!!!!!

**Comment by krishdevdb** [comment]
I am done @runningfromcyberstalker. Check #2215

File: topics/netlify/index.md (hunk 1 above)
Comment (krishdevdb) [reply topics/netlify/index.md:10]: @BlakeWilliams Kindly Check #2215. I have made the changes

**Comment by krishdevdb** [comment]
> Nothing happen

@cgwizard796 Kindly Check #2215. I have made the changes

File: topics/netlify/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: Netlify
+github_url: https://github.com/netlify
+logo: netlify.png
+short_description: Netlify: Develop & deploy the best web experiences
```
Comment (kenyonj) [inline topics/netlify/index.md:5]:
```suggestion
short_description: Develop & deploy the best web experiences
```

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Review by kenyonj — CHANGES_REQUESTED** [review]
The image needs to be 288px square and is currently 512px square. Please fix that and we can get this merged.

**Comment by kenyonj** [comment]
closing in favor of https://github.com/github/explore/pull/2215

**Pull request closed without merging**